### DISCUSSION TITLE
Incorporación de código HTML a los textos en QGIS

### DISCUSSION THREAD

**Discussion by ambarja** [qgispe/discusiones#3] [Ideas]
Esta discusión salió gracias al comentario de **Henry**
![image](https://user-images.githubusercontent.com/23284899/118148665-698aa800-b3d6-11eb-96dd-e1c66a98c961.png)

Frente a esto una posible alternativa es la siguiente: :point_down: 
```{html}
<h1 align='center' style="font-family:'Roboto'; ;color:black ;text-shadow: 3px 8px yellow;" >Holi :3 </h1>

```
![Captura de pantalla de 2021-05-13 10-33-07](https://user-images.githubusercontent.com/23284899/118148950-aeaeda00-b3d6-11eb-86f4-4c7fff74d75f.png)

**Comment by hfsoriad** [comment]
Me ha servido mucho. Sin duda eres un crack. Muchas gracias.

**Comment by lefcgis** [comment]
Está perfecto, @ambarja !!!
